CONTRIBUTING.md

```
@@ -77,5 +147,12 @@ The first line is the subject and should be no longer than 70 characters, the se
 [operator_ecosystem_contact]: https://groups.google.com/g/red-hat-operator-ecosystem
 [operator_ecosystem_org]: https://github.com/redhat-openshift-ecosystem
 [developer_docs]: https://github.com/redhat-openshift-ecosystem/openshift-preflight/tree/main/docs/dev
+[library_docs]: https://github.com/redhat-openshift-ecosystem/openshift-preflight/tree/main/docs/LIBRARY.md
+[dev_setup_docs]: https://github.com/redhat-openshift-ecosystem/openshift-preflight/tree/main/docs/dev/README.md#dev-setup
+[recipes_docs]: https://github.com/redhat-openshift-ecosystem/openshift-preflight/tree/main/docs/RECIPES.md
+[index_image_docs]: https://github.com/redhat-openshift-ecosystem/openshift-preflight/tree/main/docs/BUILDING_AN_INDEX.md
+[testing_docs]: https://github.com/redhat-openshift-ecosystem/openshift-preflight/tree/main/docs/dev/TESTING.md
+[implement_check_docs]: https://github.com/redhat-openshift-ecosystem/openshift-preflight/tree/main/docs/dev/IMPLEMENT_A_CHECK.md
+[errors_logging_docs]: https://github.com/redhat-openshift-ecosystem/openshift-preflight/tree/main/docs/dev/ERRORS_AND_LOGGING.md
 [reporting_issues]: https://sdk.operatorframework.io/docs/contribution-guidelines/reporting-issues/
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "reporting_issues"

(MD053, link-image-reference-definitions)